### PR TITLE
docs

### PR DESCRIPTION
**Pull request by odscjames**

### CODE REVIEW

**Comment by odscjames** [comment]
https://json-data-ferret.readthedocs.io/en/docs/reference/index.html

**Comment by odscjames** [comment]
https://json-data-ferret.readthedocs.io/en/docs/index.html

File: docs/reference/configuration.rst [hunk 1]
```diff
@@ -42,7 +45,7 @@ The value should be the actual JSON Schema as a python dictionary. You will prob
 This is optional; if not set basic operations will still work.
 
 Spreadsheet Guide Form
-----------------------
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (Lathrisk) [inline docs/reference/configuration.rst:48]: `<rant> I really dislike RST headers </rant>`

File: docs/how-to-guide/use-as-a-library-in-your-app.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.. code-block:: bash
+
+    urlpatterns = [
+        ...
+        path("jsondataferret/", include("jsondataferret.urls")),
+        ...
+    ]
+
+For each user you want to be able to use this web UI, you need to give them the correct permission. You can do this by any usual Django means - logging in to admin interface as a super user is probably easiest.
```
Comment (Lathrisk) [inline docs/how-to-guide/use-as-a-library-in-your-app.rst:56]: This seems quite wordy, perhaps something like:
You need to set the correct permissions for each user of the web UI. You can do this by any Django allows - e.g. logging into the admin interface as a superuser.
Or 
To allow new users to use the web UI, you must first set the correct permissions. You can set these permissions by using Django, e.g. logging into the interface as a superuser.

File: docs/how-to-guide/use-as-a-library-in-your-app.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Use from your custom code
+-------------------------
+
+You can now use the Python API and read the models of this library as you require.
+:doc:`See the Python API reference for more <../reference/python-apis>`
+
+Web UI
+------
+
+If you want people to be able to use this, you must enable this and give user accounts permission.
```
Comment (Lathrisk) [inline docs/how-to-guide/use-as-a-library-in-your-app.rst:44]: Perhaps too many `this`s. 
`If you want people to be able to use the Web UI, you must first enable it and set the correct user account permissions`.

File: docs/how-to-guide/use-as-a-library-in-your-app.rst (hunk 2 above)
Comment (odscjames) [reply docs/how-to-guide/use-as-a-library-in-your-app.rst:56]: Also realised it doesn't say what the correct permission is, I'll fix that too.

File: docs/how-to-guide/use-as-a-library-in-your-app.rst (hunk 3 above)
Comment (odscjames) [reply docs/how-to-guide/use-as-a-library-in-your-app.rst:44]: Also added: the **relevant** user accounts permission.

**Review by Lathrisk — APPROVED** [review]

**Pull request merged**